### ISSUE TITLE
Issue about chexpert plus dataset

### ISSUE THREAD

**Issue by Yingshu-Li** [Event-AHU/Medical_Image_Analysis#28]
Hi,

As Chexpert Plus has its official split, training, and validation. Why do you do another random split?

And I found in your split, there are some original training data in your test split.

**Comment by QwYko-AHU** [comment]
Our division principle follows the approach of R2GenCSR. The result of the division is that the training set: validation set: test set =40463:5780:11562. This is a clerical error in the R2GenCSR paper.

**Comment by QwYko-AHU** [comment]
> > Our division principle follows the approach of R2GenCSR. The result of the division is that the training set: validation set: test set =40463:5780:11562. This is a clerical error in the R2GenCSR paper.
> 
> Thanks for your reply.
> 
> Is there any reason that you re-split the dataset instead of using the official train-val split of ChexpertPlus？

The official classification only relied on what was observed during the inspection (or the inspection impression). There were only 60,000 pairs of data. However, we used both the observed results and the inspection impression, which resulted in a data volume of over 200,000. So we have newly divided the training set and the test set.
